test/e2e/pageObjects/compliance/checkMrn.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const urls = require('urls');
+
+function goToCheckMrnPage(mrnDate) {
+
+    const I = this;
+
+    I.fillField('#MRNDate_day', mrnDate.date());
+    I.fillField('#MRNDate_month', mrnDate.month() + 1);
+    I.fillField('MRNDate_year', mrnDate.year());
+    I.click('input[type="submit"]');
+    I.seeInCurrentUrl(urls.compliance.checkMRNDate);
+
+}
+
+function goToCorrectPageAfterCheckMRN(value, url) {
+
+    const I = this;
+
+    I.checkOption(`#CheckMRN_checkedMRN-${value}`);
+    I.click('input[type="submit"]');
```

Comment:
I much prefer to use I.click('Continue'); as it reads a lot easier. It there a good reason to change it?

Reply:
Not sure why I didn't put Continue! Will change it now